examples/bitcoin_psbt/test/manual_tests.ts

```diff
@@ -0,0 +1,12 @@
+import { getCanisterId } from 'azle/dfx';
+import { runTests } from 'azle/test';
+
+import { getTests } from './tests';
+
+const canisterId = getCanisterId('basic_bitcoin');
```

Comment:
Let's rename the canister to bitcoin_psbt

Reply:
done